src/script/E2EIdentity/E2EIdentityEnrollment.ts

```diff
@@ -54,7 +61,11 @@ type EnrollmentConfig = {
   discoveryUrl: string;
   gracePeriodInMs: number;
 };
+
+const gracePeriodMS = 7 * TimeInMillis.DAY; //GP
```

Comment:
I'm a little confused what is this graceperiod compared to the grace period that is given in the `initialize` function 🤔 . 

Maybe the naming needs to be improved there

Reply:
I will be using the same gracePeriod thats coming from TM so this will be removed.